__tests__/app.test.js

```diff
@@ -209,4 +222,102 @@ describe("/api/articles/:article_id/comments", () => {
         expect(body.msg).toBe("Bad Request");
       });
   });
+  test("POST: 400 sends status and error message when given an invalid id", () => {
+    const commentToSend = {
+      username: "butter_bridge",
+      body: "This is a new comment",
+    };
+    return request(app)
+      .post("/api/articles/notAnArticle/comments")
+      .send(commentToSend)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+  test("POST: 404 when given invalid endpoint", () => {
+    const commentToSend = {
+      username: "butter_bridge",
+      body: "This is a new comment",
+    };
+    return request(app)
+      .post("/api/articles/1/notComments")
+      .send(commentToSend)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("path does not exist");
+      });
+  });
+  test("POST: 404 sends status and error message when given a valid but non-existent id", () => {
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("article does not exist");
+      });
+  });
+});
+
+// QUESTION 8
+describe("/api/articles/:article_id", () => {
+  test("PATCH: 200 responds with updated article with the votes incremented", () => {
+    const patchToSend = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/2")
+      .send(patchToSend)
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
I would still show that you get all the other properties but the values can be the data type